### PR TITLE
Move input into the container

### PR DESCRIPTION
**Pull request by hhorak**
Otherwise it does not work in podman 1.6.x due to https://bugzilla.redhat.com/show_bug.cgi?id=1827324

### CODE REVIEW

File: test/run_test [hunk 1]
```diff
@@ -872,19 +875,19 @@ run_pgaudit_test()
   wait_ready "$name"
 
   # enable the pgaudit extension
-  docker exec -i $(get_cid "$name") bash -c psql <<EOSQL
+  docker exec -i $(get_cid "$name") bash -c "psql <<EOSQL
```
Comment (praiskup) [inline test/run_test:878]: Hehe, what a hack :-).  Don't you want to add inline coment with the reason why the
heredoc separator went into the string? (rhbz# reference at least).

Comment (hhorak) [reply test/run_test:880]: Good idea, done.

**Comment by pkubatrh** [comment]
There are still some heredocs where the same is needed. Eg.
https://github.com/sclorg/postgresql-container/blob/master/test/pg-test-lib.sh#L61

**Comment by hhorak** [comment]
> There are still some heredocs where the same is needed. Eg.
> https://github.com/sclorg/postgresql-container/blob/master/test/pg-test-lib.sh#L61

Fixed now as well. Although the biggest challenge was work-arounding the `docker exec ... <file` in https://github.com/sclorg/postgresql-container/blob/master/test/pagila.sh#L37. Let's see whether this works now.

**Comment by pkubatrh** [comment]
lgtm now, thanks!

**Pull request merged**